src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -459,6 +489,12 @@ object FunctionRegistry {
     bind(name, MethodType(hrt.typ, hru.typ, hrv.typ, hrw.typ), Arity3Aggregator[T, U, V, W, X](hrx.typ, ctor))
   }
 
+  def registerAggregator[T, U, V, W, X, Y, Z](name: String, ctor: (U, V, W, X, Y) => TypedAggregator[Z])
```

Comment:
I'm confused why you had to add this.  It doesn't look like you register an arity 5 aggregator anywhere.